ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/NodeKernel.hs

```diff
@@ -239,10 +261,9 @@ forkBlockForging
        (IOLike m, RunNode blk)
     => InternalState m addrNTN addrNTC blk
     -> BlockForging m blk
-    -> m ()
+    -> m (Thread m Void)
 forkBlockForging IS{..} blockForging =
-      void
-    $ forkLinkedWatcher registry threadLabel
+    forkLinkedWatcher registry threadLabel
```

Comment:
May we use this opportunity to also label this thread?

Reply:
doesn't `threadLabel` argument do that?

Reply:
Yeah it is already labelled